modules/otel-collector-components.adoc

```diff
@@ -425,15 +515,15 @@ The Resource processor applies changes to the resource attributes. This processo
 Attributes represent the actions that are applied to the resource attributes, such as delete the attribute, insert the attribute, or upsert the attribute.
 
 [id="span-processor_{context}"]
-=== Span processor
+=== Span Processor
 
-The Span processor is currently a link:https://access.redhat.com/support/offerings/techpreview[Technology Preview] feature only.
+The Span Processor is currently a link:https://access.redhat.com/support/offerings/techpreview[Technology Preview] feature only.
 
-The Span processor modifies the span name based on its attributes or extracts the span attributes from the span name. It can also change the span status. It can also include or exclude spans. This processor supports traces.
+The Span Processor modifies the span name based on its attributes or extracts the span attributes from the span name. It can also change the span status. It can also include or exclude spans. This processor supports traces.
```

Comment:
```suggestion
The Span Processor modifies the span name based on its attributes or extracts the span attributes from the span name. It can also change the span status, and include or exclude spans. This processor supports traces.
```

Reply:
Good to have ..

Reply:
Thank you, applying.